app/client/ui/DescriptionConfig.ts

```diff
@@ -11,8 +11,9 @@ const t = makeT('FieldConfig');
 
 export function buildDescriptionConfig(
     owner: MultiHolder,
-    origColumn: ColumnRec,
+    parent: ColumnRec | ViewSectionRec,
```

Comment:
```suggestion
    description: KoSaveableObservable<string>,
```

Since we really only need the `description` here, we can change the parameter so that the callers have to pass in the description.